source/containers.tex

```diff
@@ -700,6 +700,31 @@
 
 \end{libreqtab4a}
 
+\pnum
+In this Clause the behavior of certain member functions and deduction guides
```

Comment:
...`and Clause~\ref{strings}`. Also need to update the cross reference in [string.cons] after the move.